FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -35,11 +35,14 @@ final class EncryptedStorage: EncryptedStorageType {
     // new schema should be added as a new case
     private enum EncryptedStorageSchema: CaseIterable {
         case initial
+        case version5
 
         var version: SchemaVersion {
             switch self {
             case .initial:
                 return SchemaVersion(appVersion: "0.2.0", dbSchemaVersion: 4)
+            case .version5:
+                return SchemaVersion(appVersion: "0.2.0", dbSchemaVersion: 5)
```

Comment:
@tomholub 
Is `appVersion` OK?

Reply:
It's outdated. Are we supposed to keep it up to date? (and for what purpose?) If yes, could you put some dynamic value in there?

Reply:
We don't use this property in any logic